client/src/components/dashboard/Team.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+
+import DashboardHeader from './DashboardHeader';
+
+import _workers from '../../constants/workers.json';
+
+function Member({ name }) {
+  return (
+    <p className='p-2 flex items-center border-b border-gray-400 font-light text-sm'>
+      <AccountCircleIcon className='mr-4' />
+      {name}
```

Comment:
Allow users to email a team member until Direct Messaging has been implemented

Reply:
Allow managers & admins to view a worker's profile info onClick in `dashboard--workers`

Reply:
To be implemented in `ft/fullstack`